### PR TITLE
Added Cart Page and linked to Navbar

### PR DESCRIPTION
**Pull request by rutu14**
Click on the cart icon on navbar, to go to the cart page.

On Clicking on Promocode Heading, input box appears.

### CODE REVIEW

File: css/cart-page.css [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    background-color: var(--secondary-colour);
+    border: none; 
+    border-radius: 50%;
+    height: 16px;
+    width: 16px;
+}
+  
+input[type="range"]:focus::-moz-range-thumb{
+    outline: 2px solid var(--secondary-colour);
+    outline-offset: 0.125rem;
```
Comment (iRohitGaur) [inline css/cart-page.css]: this can be in px

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by Omkar-Ghate — APPROVED** [review]
Looks great! Good to merge.

File: css/cart-page.css [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    "product-grid aside"
+    "product-grid aside" 
+    "product-grid aside"; 
+}
+
+.cart-summary{
+    grid-area: aside;
+    display: flex;
+    flex-direction: column;
+    box-shadow: 0 0.2rem 0.5rem 0 var(--box-shadow), 0 2px 10px 0 var(--box-shadow);
```
Comment (Muzammil-khan710) [inline css/cart-page.css]: Try px as the values are small.. in Box-shadow

**Review by Muzammil-khan710 — APPROVED** [review]
Good to merge... Try to use px for smaller units. Units which are 10px or low

File: css/cart-page.css (hunk 1 above)
Comment (rutu14) [reply css/cart-page.css]: Sure

File: css/cart-page.css (hunk 2 above)
Comment (rutu14) [reply css/cart-page.css]: Sure

**Pull request merged**